neo/SmartContract/InteropService.NEO.cs

```diff
@@ -462,5 +466,24 @@ private static bool Iterator_Concat(ApplicationEngine engine)
             engine.CurrentContext.EvaluationStack.Push(StackItem.FromInterface(result));
             return true;
         }
+
+        private static bool Json_Deserialize(ApplicationEngine engine)
+        {
+            var json = Encoding.UTF8.GetString(engine.CurrentContext.EvaluationStack.Pop().GetByteArray());
```

Comment:
Maybe we need to handle the illegal characters consistently in smart contracts.

Reply:
`JObject.Parse` should deal with this, what do you propose?

Reply:
Yes.